src/pint_pal/utils.py

```diff
@@ -1238,7 +1238,7 @@ def check_recentness_noise(tc):
         name of the most recent available set of chains
     """
     if not tc.get_noise_dir():
-        log.warning(f"Yaml file does not have a noise-dir field (or it is unset).")
+        log.warning(f"Yaml file does not have a noise-dir field (or it is unset). Will check working directory.")
```

Comment:
Is it checking the working directory in this case though? Right under this is the line `return None, None`.

Reply:
Yeah so this is because `add_noise_to_model` will go ahead and just grab whatever's in the working directory if it's not set in the YAML, but that isn't handled here, it's in `noise_utils`

Reply:
Hmm... a bit confusing, but I guess it can't be helped at the moment.